scripts/exclude-links.txt

```diff
@@ -0,0 +1,20 @@
+http://127.0.0.1:10000/devstoreaccount1;
+http://localhost:3000/
+https://$
+https://api.github.com/repos/$
+https://blog.$
+https://discuss.$
+https://dvc.org/some.link
+https://example.com/data.txt
+https://example.com/path/to/data
+https://example.com/path/to/data.csv
+https://example.com/path/to/dir
+https://github.com/$
+https://github.com/dataversioncontrol/myrepo.git
+https://github.com/example/registry
+https://github.com/iterative/dvc.org/blob/master/public$
+https://github.com/iterative/dvc/releases/download/$
```

Comment:
Oh so it's actually matching the `$` character for cases like below?

https://github.com/iterative/dvc.org/blob/2d99864409fde25a06d6d2378fc6e24aa41cc4ac/src/components/DownloadButton/index.js#L33-L48

Reply:
Yes my regexes match only up to the literal `$`. It seems a good compromise between blacklisting false negatives and programming false negatives detection.